docs/docs/user-settings/README.md

```diff
@@ -66,41 +66,43 @@ If you've already upgraded, you'll see an option to **Manage Subscription** here
 
 ### Usage
 
-[Invocations](/pricing/#invocations) are Pipedream's billable unit, and [free users](/pricing/#developer-tier) are limited on the number of daily invocations they can run. The **Usage** section displays a chart of the daily invocations across a historical range of time to provide insight into your usage patterns.
+[Credits](/pricing/#credits) are Pipedream's billable unit, and [free users](/pricing/#developer-tier) are limited on the number of daily free credits allocated. The **Usage** section displays a chart of the daily credits across a historical range of time to provide insight into your usage patterns.
```

Comment:
"are limited on the number of daily free credits allocated" -> "have a daily credit limit".